src/components/molecules/search-results-pager/search-results-pager.jsx

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { i18n } from '../../../utils';
+
+import './search-results-pager.scss';
+
+const SearchResultsPager = ({
+	current,
+	total,
+	testid = 'tid-results-pager',
+	keyword,
+	resultsPerPage,
+	isTop = true,
+	language = 'en',
+}) => {
+	// Pagination Button
+	const PgButton = (i) => {
+		let bstate = `pager__button `;
+		if (i === current) bstate += 'active ';
+		if (i === total) bstate += 'total_pages';
+		
+		return (
+			<li className="pager__list-item" key={`pager__button-${i}`}>
+				<a
```

Comment:
if the link is current shouldn't it not be a link?
Also, I'm not a huge fan of using aria label here.  There should rather be some screenreader only text that says "go to page".  In this case the screen reader reads, "Go to page 6, 6" because of the link text.  Remember, the first rule of aria is to not use aria.